src/lib/helpers.js

```diff
@@ -21,6 +21,17 @@ export const urlify = stringValue => {
   return 'undefined'
 }
 
+export const deurlify = stringValue => {
+  if (stringValue) return stringValue.split('_').join(' ')
+  return 'undefined'
```

Comment:
Why `undefined` instead of undefined?
Also, having it the other way around would make sense (handling precondition before doing function work)

Reply:
I wanted the function to always return a string. the normal undefined value renders to an empty string if used on ui.

undefined actually should only happen, if a page doesn't have a category assigned. and we should check that on generating the jsonfeed. therefor this case should only happen on very rare conditions.